lib/src/widgets/slidable.dart

```diff
@@ -341,6 +341,10 @@ class SlidableController {
 
   bool _isSlideOpen;
 
+  Function(double) onSliding;
```

Comment:
It would be better if we use the ValueChanged<double> typedef. More over, we should add a documentation for all the public apis.